conf/base.config

```
@@ -26,6 +26,11 @@ process {
     memory = { check_max( 64.GB * task.attempt, 'memory') }
     time = { check_max( 36.h * task.attempt, 'time') }
   }
+  withName:fastqc {
```

Trying to trigger the tests again.
```suggestion
  withName:fastqc { 
```